scenarios/ocd/CreateAKSDeployment/README.md

```diff
@@ -25,7 +31,7 @@ Results:
 
 ```expected_similarity=0.3
```

Comment:
The expected similarity should be declared above the codeblock using an HTML comment: `<!-- expected_similarity=0.3 -->` and you should also add the `json` annotation to the codeblock so that the expected results can compared more accurately